### PR TITLE
Emerald: put test data in structs

### PR DESCRIPTION
**Pull request by pro-wh**
for #166

### CODE REVIEW

**Review by mitjat — APPROVED** [review]
:+1: 
Thank you!

File: analyzer/uncategorized/evm_raw_tx_test.go [hunk 1]
```diff
@@ -130,29 +137,37 @@ func TestDecodeBasic(t *testing.T) {
...
-		// "cow" test account
-		"cd2a3d9f938e13cd947ec05abc7fe734df8dd826",
-		0,
+		&callDetails{
+			commonDetails: commonDetails{
+				chainID:  0,
+				value:    10_000_000_000_000_000,
+				gasLimit: 10_000,
+				gasPrice: 1_000_000_000_000,
+				// "cow" test account
```
Comment (Andrew7234) [inline analyzer/uncategorized/evm_raw_tx_test.go:146]: lol what's cow?

**Review by Andrew7234 — APPROVED** [review]

File: analyzer/uncategorized/evm_raw_tx_test.go (hunk 1 above)
Comment (aefhm) [reply analyzer/uncategorized/evm_raw_tx_test.go:146]: Yeah, have I been doing it wrong all this time with Alice/Bob? 🤔

Comment (pro-wh) [reply analyzer/uncategorized/evm_raw_tx_test.go:146]: https://github.com/oasisprotocol/oasis-sdk/blob/5aa39a2726f196a7b7c08e322ab27bf91f2670d8/runtime-sdk/modules/evm/src/raw_tx.rs#L304 ->
https://github.com/ethereum/tests/blob/v10.0/BasicTests/txtest.json ->
https://github.com/ethereum/tests/blob/v10.0/BasicTests/keyaddrtest.json#L5

saw it in a test from ethereum

interestingly, it has had quite some activity on Ethereum https://etherscan.io/address/0xcd2a3d9f938e13cd947ec05abc7fe734df8dd826

**Pull request merged**